plugins/woocommerce/client/admin/client/core-profiler/index.tsx

```diff
@@ -473,26 +469,23 @@ const updateBusinessInfo = fromPromise(
 			payload: BusinessInfoPayload;
 		};
 	} ) => {
-		const refreshedOnboardingProfile = ( await resolveSelect(
```

Comment:
the API merges the fields into the existing object so no need to fetch & merge on the frontend. This was causing consistency issues due to stale values in the fetch